pkg/controller/catalog/injector/binding_injector.go

```diff
@@ -17,15 +17,16 @@ limitations under the License.
 package injector
 
 import (
-	model "github.com/kubernetes-incubator/service-catalog/model/service_controller"
+	sbmodel "github.com/kubernetes-incubator/service-catalog/model/service_broker"
+	"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog"
 )
 
 // BindingInjector defines the interface for injecting binding credential
 // information for consumption.
 type BindingInjector interface {
 	// Inject injects binding credential information for consumption.
-	Inject(*model.ServiceBinding) error
+	Inject(*servicecatalog.Binding, *sbmodel.Credential) error
```

Comment:
That wasn't pulled out as part of this PR, but I'll take a crack at answering.

The `pkg/apis/servicecatalog` package is all k8s types. Those types all pretty much decompose into "spec" and "status," with the former being the user-defined specification of a resource and the latter being, obviously, system-updated attributes that reflect a resource's current state. It's not clear to me that (from a k8s perspective) credentials fit cleanly into one of those two categories. Think of it this way... if you type the following command, you don't actually want to _see_ credentials, right? They should only appear, properly encoded, in a secret or configmap.

```
$ kubectl get servicebinding my-binding
```